presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -1285,11 +1252,17 @@ VeloxQueryPlanConverter::toVeloxQueryPlan(
 
       if (auto constantExpr =
               std::dynamic_pointer_cast<const ConstantTypedExpr>(expr)) {
+        if (constantExpr->type()->isRow()) {
```

Comment:
Why we can't support row type and it can't be handled by columnVector->copy below as for ARRAY and MAP? Thanks!

Reply:
+1
cc @mbasmanova 
Also, why error message does not specify something like "ROW is not supported"?

Reply:
Sorry, a left over from some earlier experiment. I'm going to remove it. VALUES queries with structs do not work yet and require more changes.